### DISCUSSION TITLE
Bicep_Linter generates a json file for each of my bicep files

### DISCUSSION THREAD

**Discussion by JonasCordsen** [oxsecurity/megalinter#2127] [Q&A]
Bicep_Linter generates a json file for each of my bicep files
Is there a way for it not to do this, or at least to put it in a temp directory?

**Comment by nvuillam** [comment]
For the moment the linter is called file by file, that explains the behavior
Can bicep linter take a list of files as parameters ?

**Reply by JonasCordsen** [reply]
Sadly no.
Is it correct that megaliter does someting like a build on each file?

**Reply by nvuillam** [reply]
For linters that does not allow a list of files as input, or can not run in a single call from the root repository, sadly we need to call it once by file :/

**Reply by JonasCordsen** [reply]
It is fine that it needs to run pr file, ie issue is that when I run locally, I get a json file for each bicep file, I then need to cleanup/remove after.
is it possible to make sure that it either create those files in some temp location, or that they are removed after test are completed

**Reply by nvuillam** [reply]
How are these json files generated ?
You could:
- add their patterns to .gitignore
- call MegaLinter with `REPORT_OUTPUT_FOLDER: none` (use beta, a fix has not been released yet to an official version)
